internals/scripts/setup.js

```diff
@@ -25,7 +25,6 @@ cleanRepo(function () {
     interval = animateProgress('Installing dependencies');
   }, 500);
 
-  process.stdout.write('Installing dependencies');
```

Comment:
This was removed as it appears as a duplicate as shown below:
`Installing dependencies... (This might take a while)Installing dependencies`